src/power_grid_model/core/power_grid_model.py

```diff
@@ -282,55 +282,63 @@ def calculate_power_flow(
-                                data[indptr[k]:indptr[k + 1]]
-                                This is the concept of compressed sparse structure
-                                https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_matrix.html
-                            data: 1D numpy structured array in flat
-            threading:
-                only applicable for batch calculation
-                < 0 sequential
-                = 0 parallel, use number of hardware threads
-                > 0 specify number of parallel threads
-            output_component_types: list or set of component types you want to be present in the output dict.
-                By default all component types will be in the output
-            continue_on_batch_error: if the program continues (instead of throwing error) if some scenarios fails
+            symmetric (bool): Whether to perform a three-phase symmetric calculation.
 
-        Returns:
-            dictionary of results of all components
-                key: component type name to be updated in batch
-                value:
-                    for single calculation: 1D numpy structured array for the results of this component type
-                    for batch calculation: 2D numpy structured array for the results of this component type
-                        Dimension 0: each batch
-                        Dimension 1: the result of each element for this component type
-            Error handling:
-                in case an error in the core occurs, an exception will be thrown
-        """
+                - True: Three-phase symmetric calculation, even for asymmetric loads/generations.
+                - False: Three-phase asymmetric calculation.
+
+            error_tolerance: Error tolerance for voltage in p.u., applicable only when iterative=True.
+
+            max_iterations: Maximum number of iterations, applicable only when iterative=True.
+
+            calculation_method (an enumeration or string): The calculation method to use.
+
+                - Newton_raphson: Use Newton-Raphson iterative method (default).
+                - Linear: Use linear method.
+           
+            update_data:
+              None: Calculate power flow once with the current model attributes.
+              Or a dictionary for batch calculation with batch update.
+                  
+                    key: Component type name to be updated in batch.
+                        - For homogeneous update batch (a 2D numpy structured array):
+
+                            - Dimension 0: Each batch.
+                            - Dimension 1: Each updated element per batch for this component type.
+                        - For inhomogeneous update batch (a dictionary containing two keys):
+
+                            - indptr: A 1D integer numpy array with length n_batch + 1. Given batch number k, the update array for this batch is
```

Comment:
as you can see with this failed action: https://github.com/PowerGridModel/power-grid-model/actions/runs/5419899136/jobs/9853457656

```
 /opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/power_grid_model/core/power_grid_model.py:313:0: C0301: Line too long (144/120) (line-too-long)
```
Just put a newline before the word `update` and continue line `314` from there. that should resolve this issue. you can verify this on your own PC by running `pylint src`